specification/maps/data-plane/Creator/preview/2022-01-01-preview/examples/features/Features_GetCollection.json

```diff
@@ -0,0 +1,34 @@
+{
+  "parameters": {
+    "geography": "us",
+    "api-version": "2022-01-01-preview",
+    "datasetId": "218fda98-e638-0edf-5ef7-28491ff3bed4",
+    "collectionId": "unit"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "name": "unit",
+        "description": "A physical and non-overlapping area which might be occupied and traversed by a navigating agent. Can be a hallway, a room, a courtyard, etc. It is surrounded by physical obstruction (wall), unless the isOpenArea attribute is equal to true, and one must add openings where the obstruction shouldn't be there. If isOpenArea attribute is equal to true, all the sides are assumed open to the surroundings and walls are to be added where needed. Walls for open areas are represented as a lineElement or areaElement with isObstruction equal to true.",
+        "ontology": "Facility-2.0",
+        "links": [
+          {
+            "href": "https://us.atlas.microsoft.com/wfs/dataset/218fda98-e638-0edf-5ef7-28491ff3bed4/collections/unit/definition?api-version=2022-01-01-preview",
```

Comment:
WFS? What is it? :)

Reply:
Aghh looks like I missed some references in the examples. Thanks for catching it! Fixed ✔️